### PR TITLE
[updating html, stock analysis]

### PR DESCRIPTION
**Pull request by JasleenShergill**
[updating html, stock analysis]

### CODE REVIEW

File: Images/Emad.png
Comment (Davjes15) [inline Images/Emad.png:1]: Organize better your resources in the repo.

File: about.html
Comment (Davjes15) [inline about.html:1]: same for the html files. Your repor needs a better structure in terms of files and directories

File: style.css
Comment (Davjes15) [inline style.css:1]: same style.css should be inside a folder cold css

**Review by Davjes15 — COMMENTED** [review]
Overall it looks good but you need to improve the organization of your repo

**Comment by JasleenShergill** [comment]
files are now organized in respective folders

**Pull request merged**